hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -283,6 +318,10 @@ case class IEmitCodeGen[+A](Lmissing: CodeLabel, Lpresent: CodeLabel, value: A)
   def get(cb: EmitCodeBuilder, errorMsg: String = "expected non-missing"): A =
     handle(cb, cb._fatal(errorMsg))
 
+  def getNeverMissing(cb: EmitCodeBuilder): A = {
+    handle(cb, cb._fatal(s"Internal compiler error, unexpected value was missing."))
+  }
```

Comment:
No, this is dumb. I don't know why I didn't implement it in terms of `get`, and the only reason I added it in the first place is because I wanted to stop typing out error messages. But `get` has a default message which handles this. I'll remove.